functions/ocr/ocr-translate-text/src/main/java/functions/eventpojos/PubSubMessage.java

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package functions.eventpojos;
+
+// [START functions_helloworld_pubsub_message]
+// This POJO is based on the `PubsubMessage` object described here:
+// https://github.com/googleapis/googleapis/blob/master/google/pubsub/v1/pubsub.proto
+
+import java.util.Map;
+
+public class PubSubMessage {
```

Comment:
```suggestion
public class PubsubMessage {
```
To match other documentation and protos

Reply:
Done.